project/profiles/capacity4more/modules/c4m/restful/c4m_restful/plugins/restful/C4mRestfulEntityBaseNode.class.php

```diff
@@ -21,6 +21,12 @@ protected function checkEntityAccess($op, $entity_type, $entity) {
     $resource_name = $this->getResourceName();
     $group_id = $this->request['group'];
     $wrapper = entity_metadata_wrapper('node', $group_id);
+
+    if (empty($wrapper->c4m_og_status)) {
+      // Group ID is not being passed to RESTful API, cannot determine status.
+      return TRUE;
```

Comment:
@ybaras 
This one seems wrong, why should we give a bypass if  status field is missing ?